src/ray/gcs/gcs_server/gcs_server.cc

```diff
@@ -598,6 +601,13 @@ void GcsServer::InitGcsTaskManager() {
   rpc_server_.RegisterService(*task_info_service_);
 }
 
+void GcsServer::InitMonitorServer() {
```

Comment:
Any reason why we call it "MonitorServer" that's different from other modules' naming convention (*Manager)

Reply:
Mostly because it's not actually "managing" anything. Most of the other managers actually manage some state.